### PR TITLE
Supervise internal ownership pools

### PR DESCRIPTION
**Pull request by fishcakez**

### CODE REVIEW

File: lib/db_connection/ownership/pool.ex [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    Supervisor.start_link(children, sup_opts)
+  end
+
+  def pool_pid(pool) do
+    children = Supervisor.which_children(pool)
+    case List.keyfind(children, :pool, 0) do
+      {_, pool, _, _} when is_pid(pool) ->
+        pool
+      _ ->
+        IO.inspect children
```
Comment (josevalim) [inline lib/db_connection/ownership/pool.ex]: Missing inspect.

**Comment by josevalim** [comment]
Looks good but there is a failure in the build :S

File: lib/db_connection/ownership/pool.ex (hunk 1 above)
Comment (fishcakez) [inline lib/db_connection/ownership/pool.ex]: Thanks

**Comment by fishcakez** [comment]
> Looks good but there is a failure in the build :S

I had a few more failures locally. I will post them in an issue.

**Comment by fishcakez** [comment]
merged manually.

**Pull request closed without merging**